benchmarks/simple_rma.cpp

```diff
@@ -0,0 +1,342 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2020, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#include <iostream>
```

Comment:
Comment on what this benchmark is doing. This seems some kind of complete computation..